### ISSUE TITLE
clientHeight error spam

### ISSUE THREAD

**Issue by singatias** [MonoLizard/ngx-toastit#4]
Hi,

everytime we add a toast we get the following error in the console. Every thing works as intended but the error spam is annoying, I've tried to isolate the issue by removing all options except for title and it's still spam the same error.
```
ERROR TypeError: Cannot read property 'clientHeight' of null
    at ToastitComponent.ngAfterViewInit (ngx-toastit.js:81)
    at callProviderLifecycles (core.js:28921)
    at callElementProvidersLifecycles (core.js:28886)
    at callLifecycleHooksChildrenFirst (core.js:28868)
    at checkAndUpdateView (core.js:39119)
    at callWithDebugContext (core.js:40449)
    at Object.debugCheckAndUpdateView [as checkAndUpdateView] (core.js:40032)
    at ViewRef_.detectChanges (core.js:27807)
    at ApplicationRef.tick (core.js:36111)
    at core.js:35959
```

EDIT: I found why, when message is null in the option the error appear

src/lib/toastit.component.ts
```typescript
ngAfterViewInit() {
        this.viewHeight = this.elementRef.nativeElement.clientHeight;
        /** @type {?} */
        const titleElement = this.elementRef.nativeElement.querySelector('.toastit__title');
        /** @type {?} */
        const messageElement = this.elementRef.nativeElement.querySelector('.toastit__message'); <---- this can be null
        /** @type {?} */
        const titleHeight = titleElement.clientHeight;
        /** @type {?} */
        const messageHeight = messageElement.clientHeight;
        titleElement.style['max-height'] = titleHeight + 'px';
        messageElement.style['max-height'] = messageHeight + 'px';
        this.animateIn();
    }
```

here is a quick fix.
```typescript
public ngAfterViewInit() {
        this.viewHeight = this.elementRef.nativeElement.clientHeight;
        const titleElement = this.elementRef.nativeElement.querySelector('.toastit__title');
        const messageElement = this.elementRef.nativeElement.querySelector('.toastit__message');
        const titleHeight = titleElement.clientHeight;
        if (messageElement)
            messageElement.style['max-height'] = messageElement.clientHeight + 'px';

        titleElement.style['max-height'] = titleHeight + 'px';
        this.animateIn();
    }
```

**Comment by MonoLizard** [comment]
Hi @mathias-poweredsoft yeah you are right, I will fix it right away

**Comment by MonoLizard** [comment]
Fixed in v1.0.2

**Comment by singatias** [comment]
awesome, thanks :)
